ckanext/resource_type_validation/resources/resource_types.json

```diff
@@ -45,7 +45,7 @@
     "extra_mimetypes": {
         ".accdb": "application/msaccess",
         ".asc": "application/x-ascii-grid",
-        ".ecw": "application/x-ecw",
+        ".ecw": "application/octet-stream",
```

Comment:
should this be both types just in case?

Reply:
It can only be one type, although it's possible for `application/x-ecw` to override `application/octet-stream` (and indeed, that's the default; anything is allowed to override octet-stream).

I'm not sure where I originally picked up `application/x-ecw` from, but Wikipedia and IANA and several other sources don't seem to list any MIME type for ECW files at all. Our sniffing tools don't suggest anything except `application/octet-stream`, anyway, and as long as that's the case, there isn't any real risk in using that. It would only be a problem if content sniffing started to actually recognise the files as a specific type - but even if that were the case, there's no guarantee that that type would be `application/x-ecw`. I think it's okay as-is.